hr_employee_phone_pin/views/hr_employee_views.xml

```diff
@@ -6,8 +6,14 @@
         <field name="inherit_id" ref="hr.view_employee_form" />
         <field name="arch" type="xml">
             <field name="mobile_phone" position="after">
-                <field name="mobile_phone_pin" groups="hr.group_hr_user" />
-                <field name="mobile_phone_puk" groups="hr.group_hr_user" />
+                <field
+                    name="mobile_phone_pin"
+                    groups="hr_employee_phone_pin.group_mobile_phone_pin_manager"
+                />
+                <field
+                    name="mobile_phone_puk"
+                    groups="hr_employee_phone_pin.group_mobile_phone_pin_manager"
+                />
```

Comment:
I'm wondering if maybe there should be an additional security check to allow the employee him/herself at least read access to this field.

Reply:
Well, Actually, employees don't have access to hr.employee view, so we should need to do something different :thinking: However, I think that this info is more related to IT

Reply:
> Well, Actually, employees don't have access to hr.employee view, so we should need to do something different 🤔 However, I think that this info is more related to IT

@mtelahun @etobella I´ve reviewed the approach in order to make the PIN/PUK visible from the hr.employee.public form, in a way that the employee can only see its own PIN/PUK, and only if it belongs to a specific group (group_mobile_phone_pin_user). I´ve done it this way because hr.employee form view is only available for hr.group_hr_user group users, or any other group which inherits hr.group_hr_user.

On the other hand I understand that this info is related to IT in several ways, but unless a new group inherits hr.group_hr_user the hr.employee form wont be visible (at least I don't know hot to make it available 😅 with limited permissions), which in my opinion shows way too much private information for any user outside HR. Also, in this particular scenario we need the feature without having maintenance involved.

Any ideas are appreciated 👌